File: src/SOP8/Base/Account/Base.Account.IBLL/Response/ResponseOption.cs
using Response;
using System.Collections.Generic;

namespace Base.Account.IBLL.Response
{
    using Models;

    public class ResponseOption : MessageResult
    {
        private int m_nUserNo = -1;
        private List<UserOption> m_options = null;

        public int UserNo
        {
            get { return m_nUserNo; }
            set { m_nUserNo = value; }
        }

        public List<UserOption> Options
        {
            get { return m_options; }
            set { m_options = value; }
        }

        public ResponseOption()
            : base()
        {
        }

        public ResponseOption(bool success, string strMessage)
            : base(success, strMessage)
        {
        }
    }
}

File: src/SOP8/Egovframework/Base/base-account-ibll/src/main/java/egovframework/base/account/ibll/response/ResponseOption.java
package egovframework.base.account.ibll.response;

import java.util.List;
import egovframework.base.account.ibll.models.UserOption;
import egovframework.base.response.MessageResult;

public class ResponseOption extends MessageResult {

    private int userNo = -1;
    private List<UserOption> options = null;

    public int getUserNo() {
        return userNo;
    }

    public void setUserNo(int userNo) {
        this.userNo = userNo;
    }

    public List<UserOption> getOptions() {
        return options;
    }

    public void setOptions(List<UserOption> options) {
        this.options = options;
    }

    public ResponseOption() {
        super();
    }

    public ResponseOption(boolean success, String message) {
        super(success, message);
    }
}